honeybee_radiance_folder/folder.py

```diff
@@ -228,425 +354,184 @@ def daylight_matrix_files(self, aperture, receiver=None):
-        """
-        for subfolder in self.ASSET:
-            directory = os.path.join(self.folder, subfolder)
-            if os.path.exists(directory) and not overwrite:
-                raise ValueError('{} already exist.'.format(directory))
-            os.makedirs(directory)
-
-
-# TODO: Add look up for files from output folder.
-# unlike other folders output should probably be more flexible on file extension and
-# accept a list of extentions. We may need to change this globally.
-class OutputFolder(_FolderCollection):
-    """Radiance output folder.
-
-    .. code-block:: shell
 
-        ├───output                    [required]
-            ├───matrix
-            ├───octree
-            ├───postprocess
-            └───temp
-    """
-    OUTPUT_ROOT = 'output'
-    OCTREE = 'output/octree'
-    MATRIX = 'output/matrix'
-    TEMP = 'output/temp'
-    POSTPROCESS = 'output/postprocess'
-    OUTPUT = OUTPUT_ROOT, OCTREE, MATRIX, TEMP, POSTPROCESS
-    REQUIRED = (OUTPUT_ROOT,)
-
-    @property
-    def root(self):
-        """Output root folder."""
-        return self.OUTPUT_ROOT
-
-    @property
-    def octree(self):
-        """Folder for octree files."""
-        return self.OCTREE
-
-    @property
-    def matrix(self):
-        """Matrix outputs folder."""
-        return self.MATRIX
-    
-    @property
-    def temp(self):
-        """Temporary files folder."""
-        return self.TEMP
-    
-    @property
-    def postprocess(self):
-        """Postprocess folder."""
-        return self.postprocess
-
-    def write(self, overwrite=True):
-        """Write model folder.
-
-        Args:
-            overwrite (bool): Set to True to overwrite the folder is it already exist.
+        Returns:
+            path to folder.
         """
-        for subfolder in self.OUTPUT:
-            directory = os.path.join(self.folder, subfolder)
-            if os.path.exists(directory) and not overwrite:
-                raise ValueError('{} already exist.'.format(directory))
-            os.makedirs(directory)
-
+        assert 0 <= folder_type <= 2, 'folder_type must be an integer between 0-2.'
 
-class SystemFolder(_FolderCollection):
-    """System folder includes settings for runs like options, tasks.json."""
-    SYSTEM_ROOT = 'system'
-    SYSTEM = (SYSTEM_ROOT,)
-    REQUIRED = (SYSTEM_ROOT,)
+        if not cfg:
+            cfg = dict(config.minimal)
 
-    @property
-    def root(self):
-        """System root folder."""
-        return self.SYSTEM_ROOT
+        if folder_type == 0:
+            cfg['GRID'] = True
+        elif folder_type == 1:
+            cfg['VIEW'] = True
+        else:
+            cfg['GRID'] = True
+            cfg['VIEW'] = True
 
-    def write(self, overwrite=True):
-        """Write model folder.
+        root_folder = self.model_folder(full=True)
+        if overwrite:
+            shutil.rmtree(root_folder, ignore_errors=True)
```

Comment:
I think this was the method that wasn't working in most of my tests. It might be a good idea to take a look at what happens in the ladybug `nukedir` method to see some more reliable options.